modules/reference/pages/public-metrics-reference.adoc

```diff
@@ -112,6 +112,27 @@ Number of unavailable partitions (the partitions that lack quorum among their re
 
 ---
 
+[[redpanda_cluster_latest_cluster_metadata_manifest_age]]
+=== redpanda_cluster_latest_cluster_metadata_manifest_age
+
+The age in seconds since the last time Redpanda uploaded metadata files to tiered storage for your cluster. A value of `0` indicates metadata has not yet been uploaded. When performing a xref:manage:whole-cluster-restore.adoc[whole cluster restore] operation, the metadata for the new cluster will not have any changes made to the source cluster which are newer than this age.
+
+The `redpanda_namespace` label supports the following options for this metric:
+
+* `kafka` - user topics
+* `kafka_internal` - internal kafka topic such as consumer groups
+* `redpanda - redpanda-only internal data
+
+*Type*: gauge
+
+*Usage*: on a healthy system, this should not exceed the value set for `cloud_storage_cluster_metadata_upload_interval_ms`. You may consider setting an alert if this remains `0` for longer than 1.5 * `cloud_storage_cluster_metadata_upload_interval_ms` as that may indicate a configuration issue.
+
+*Labels*:
+
+* `redpanda_namespace=("kafka" | "kafka_internal" | "redpanda")`
+* `redpanda_topic`
+* `redpanda_partition`
+
```

Comment:
like the other comment:

this block should be removed.
there are no labels for this metric, there is only one instance of it per node

Reply:
Removed the labels information from this metric.